<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySQL Database Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/text2img.ico</normaloff>:/icon/text2img.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/icon/icon/cloud-sql3-14659761.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MySQL Database viewer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>1271</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1251</width>
       <height>541</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>199</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>10</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>항목</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>파트명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>패키지</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>제조사</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>771</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_getdata">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Get Data</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/icons8-download-48.png</normaloff>:/icon/icon/icons8-download-48.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exportdata">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Export Data</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/icons8-export-32.png</normaloff>:/icon/icon/icons8-export-32.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_importdata">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Import Data</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/import.png</normaloff>:/icon/icon/import.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_search">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addpart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Add Part</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/plus-button-4885419.png</normaloff>:/icon/icon/plus-button-4885419.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_editpart">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Edit Part</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/edit-button-84380.png</normaloff>:/icon/icon/edit-button-84380.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>항목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>102</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>파트명</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>102</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>패키지</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>102</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>102</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>제조사</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_item">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_partName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_package">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_qty">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_vendor">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deletepart">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Delete Part</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/delete-2550213.png</normaloff>:/icon/icon/delete-2550213.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Search</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/magnifying-glass-4426017.png</normaloff>:/icon/icon/magnifying-glass-4426017.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Reset</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/reset.png</normaloff>:/icon/icon/reset.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>441</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_host">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>14</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_user">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>54</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>USER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Bold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HOST</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_db">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>94</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pw">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>134</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>134</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Connect</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/network-connection-9261035.png</normaloff>:/icon/icon/network-connection-9261035.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>14</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
